source/includes/extracts-atlas-search-facts.yaml

```diff
@@ -0,0 +1,24 @@
+ref: fact-text-search-legacy-atlas
+content: |
+
+  MongoDB offers a :atlas:`premium full-text search solution, MongoDB 
+  Atlas Search </atlas-search/>`, for data hosted on :atlas:`MongoDB 
+  Atlas </>`. A legacy text search capability is available for users 
+  self-managing MongoDB deployments. 
```

Comment:
Can we consider a similar parallelized bulleted approach here?

```
- For data hosed on MongoDB Atlas, MongoDB offers...

- For self-managed deployments,  MongoDB offers legacy text search capability.

Reply:
I don't want to start the page with bullet points. So, created separate paragraphs. Hope this looks ok.

Reply:
That makes sense!